publish-packages.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Publishes the NuGet packages to the specified location."
+    echo "For publishing to Azure the following properties are required."
+    echo "   /p:CloudDropAccountName=\"account name\""
+    echo "   /p:CloudDropAccessToken=\"access token\""
+    echo "   /p:__BuildType=\"Configuration\""
+    echo "Configuration can be Release, Checked, or Debug"
+    exit 1
+}
+
+working_tree_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+publish_log=$working_tree_root/publish.log
+
+while [[ $# > 0 ]]
+do
+    unprocessedBuildArgs="$unprocessedBuildArgs $1"
```

Comment:
What is the difference between `unprocessedBuildArgs` and `$*`? for this particular script where there are no specific script parameter.  `unprocessedBuildArgs` is not neccesary